File: CS/DSWizardQueryTables/DataSourceWizardCustomization.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.DashboardCommon;
using DevExpress.DashboardWin.ServiceModel;
using DevExpress.DataAccess.UI.Wizard;
using DevExpress.DataAccess.Wizard.Presenters;
using DevExpress.DataAccess.Wizard.Views;

namespace DSWizardCustomizationNoSP
{
    public class DataSourceWizardCustomization : IDashboardDataSourceWizardCustomization
    {
        public void CustomizeDataSourceWizard(IWizardCustomization<DashboardDataSourceModel> customization)
        {
            customization.RegisterPage<ConfigureQueryPage<DashboardDataSourceModel>, ConfigureQueryPageEx<DashboardDataSourceModel>>();
            customization.RegisterPageView<IConfigureQueryPageView, ConfigureQueryPageViewEx>();
        }
    }
}

File: VB/DSWizardQueryTables/DataSourceWizardCustomization.vb
Imports DevExpress.DashboardCommon
Imports DevExpress.DashboardWin.ServiceModel
Imports DevExpress.DataAccess.UI.Wizard
Imports DevExpress.DataAccess.Wizard.Presenters
Imports DevExpress.DataAccess.Wizard.Views

Namespace DSWizardCustomizationNoSP
    Public Class DataSourceWizardCustomization
        Implements IDashboardDataSourceWizardCustomization

        Public Sub CustomizeDataSourceWizard(ByVal customization As IWizardCustomization(Of DashboardDataSourceModel)) _
            Implements IDashboardDataSourceWizardCustomization.CustomizeDataSourceWizard
            customization.RegisterPage(Of ConfigureQueryPage(Of DashboardDataSourceModel), 
                ConfigureQueryPageEx(Of DashboardDataSourceModel))()
            customization.RegisterPageView(Of IConfigureQueryPageView, ConfigureQueryPageViewEx)()
        End Sub
    End Class
End Namespace
